web/src/util/build.js

```diff
@@ -8,9 +8,11 @@ import config from '../app/config.js';
 const staticDir = config.staticDir;
 const require = createRequire(import.meta.url);
 // resolves to the node_modules folder with govuk-frontend and @ministryofjustice/frontend in
-const govUkNodeModules = path.resolve(require.resolve('govuk-frontend'), '../../../..');
+const govUkFrontend = path.resolve(require.resolve('govuk-frontend'), '../../../..');
 const mojNodeModules = path.resolve(require.resolve('@ministryofjustice/frontend'), '../../../..');
 
+console.log('CONFIG::>>>>', config);
```

Comment:
let's remove this, I assume it was for debugging.

Reply:
oops